src/material.h

```diff
@@ -75,4 +77,13 @@ class material_type
         const mat_burn_data &burn_data( size_t intensity ) const;
 };
 
+namespace materials
```

Comment:
> If it does how common actually is it?

Not quite common.

> It's not predictable if it's different to most of the other code.

If [any] existing code is a mess, it's not a valid argument to make the new one so. I can (will) unify the relevant code within a single (next) PR: it's a fairly small change.